.gitlab/release/tag-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euxo pipefail
+IFS=$'\n\t'
+
+echo "Grabbing GitHub deploy key and starting ssh-agent"
+set +x
+eval "$(ssh-agent -s)"
+aws ssm get-parameter --region us-east-1 --name ci.integrations-extras.github_deploy_key --with-decryption --query "Parameter.Value" --out text | ssh-add -
+set -x
+
+git remote set-url origin git@github.com:DataDog/integrations-extras.git
+git config --global user.email "$TAGGER_EMAIL"
+git config --global user.name "$TAGGER_NAME"
+
+set +e
+ddev release tag all
```

Comment:
```suggestion
ddev -e release tag all
```
Or set the default repo earlier ?

Reply:
in the yaml